src/library/Core.nim

```diff
@@ -1448,30 +1446,23 @@ proc defineLibrary*() =
             1 2 3
             b: unstack 2        ; b: [3 2]
             ..........
+            ; You can also discard the values using `discard`
             1 2 3
-            unstack.discard 1   ; popped 3 from the stack
+            discard unstack 1   ; popped 3 from the stack
         """:
             #=======================================================
-            if Stack[0..SP-1].len < x.i: Error_StackUnderflow()
-            
-            let doDiscard = (hadAttr("discard"))
+            if Stack[0..SP-1].len < x.i: 
+                Error_StackUnderflow()
             
             if x.i==1:
-                if doDiscard: discard stack.pop()
-                else: discard
+                discard
```

Comment:
I'm seriously confused here (perhaps I haven't properly woken up yet, or sth):

- why isn't that `push(stack.pop())` ?!

P.S. I'm not saying *you* did anything wrong; I'm just totally baffled how it already was... you merely eliminated the `if doDiscard` clause. We'll have to check this to make sure (and not just in the REPL 😉 )

Reply:
Ohh, alright! Sorry for delaying, I didn't noticed this message.

Reply:
> Ohh, alright! Sorry for delaying, I didn't noticed this message.

No worries! 😉 
And also, it's something we have to check... It appears obvious, but is it?

Reply:
@RickBarretto Have you had time to look into that? (I'm getting the to-do list cleaned so that we can *finally* make a release! 😉 )

Reply:
Sorry, totally forgot about this. Working right now! :wink:

Reply:
@drkameleon, basically this is due to the REPL. This works for scripts, but not for the REPL. 😉 

![image](https://github.com/user-attachments/assets/2cd890c2-c581-4ec1-80df-b1fb8eb004a0)
![image](https://github.com/user-attachments/assets/84822841-abbc-4c5d-9395-749c26ed9414)